src/main/java/seedu/ichifund/model/analytics/BalanceTrendReport.java

```diff
@@ -1,21 +1,62 @@
 package seedu.ichifund.model.analytics;
 
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.collections.ObservableList;
 import seedu.ichifund.model.Model;
+import seedu.ichifund.model.amount.Amount;
+import seedu.ichifund.model.date.Month;
+import seedu.ichifund.model.date.Year;
+import seedu.ichifund.model.transaction.Transaction;
 
 /**
  * Represents a report for balance trends.
```

Comment:
I initially wanted to implement a wealth/net worth but I haven't got around to doing it. Might consider planning for that information as well (or implementing yourself and then using the information yourself)